src/main/java/LacksDataBaseProject/AddAndRemoveLack.java

```diff
@@ -0,0 +1,45 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveLack {
+    static Scanner input = new Scanner(System.in);
+
+/*    protected void addlack(Lack lack,User user) {
+        if (checkIfForwarder(user)) {
+            lack.item = input.next();
+            System.out.println("Required amount: ");
+            lack.requiredAmount = input.nextInt();
+            System.out.println("Unit of measure: ");
+            lack.unitOfMeasure = input.next();
```

Comment:
tutaj jeszcze trzeba dołożyć dodawanie tego Suppliera z listy supplierów tak jak rozmawialiśmy, bo jak wyleciała metoda chooseSupplier to gdzieś go trzeba do tego Lack'u dodać.